### DISCUSSION TITLE
Any new dll release coming?

### DISCUSSION THREAD

**Discussion by pAnd0rASBG** [phpredis/phpredis#2328] [General]
Hi,
is there a chance to get dlls for IIS compatible to 8.2 / 8.3 any time soon? 
https://pecl.php.net/package/redis has been quiet for almost a year now.

Best regards,
Chris

**Comment by yatsukhnenko** [comment]
Hi, there is no ETA for new release. You can try to use DLLs from https://github.com/phpredis/phpredis/issues/2315#issuecomment-1375934796

**Comment by pculka** [comment]
Hi there,

was in same boat, compiled it myself for 8.2.5 x64

https://github.com/pculka/phpredis/releases/tag/5.3.7
